packages/browser-agent-core/src/features/spa/aggregate/index.js

```diff
@@ -125,6 +125,7 @@ export class Aggregate extends FeatureBase {
 
     state.initialPageLoad = new Interaction('initialPageLoad', 0, state.lastSeenUrl, state.lastSeenRouteName, onInteractionFinished, agentIdentifier)
     state.initialPageLoad.save = true
+    state.prevInteraction = state.initialPageLoad
```

Comment:
Real minor, but could you add a `= null` to the `this.state` block above?

Reply:
Why?